corehq/apps/reports/static/reports/ko/saved_reports.js

```diff
@@ -70,6 +70,9 @@ var ReportConfig = function (data) {
             } else if (date_range === 'lastn') {
                 days = self.days();
             } else {
+                if (self.report_name() === 'Case List') {
```

Comment:
does this work with translations? seems nicer if we can somehow pass this info as metadata into `self` but assume that was too hard to bother with? second fallback would be to at least use a slug instead of the readable name